beancount2ledger/cli.py

```diff
@@ -21,6 +20,12 @@
 import beancount2ledger
 
 
+# Beancount has limited Unicode support, UTF8 is the richest encoding supported (and not
+# everywhere)
```

Comment:
The beancount command line tools only support UTF8 encoded input. The parser supports other encodings for string literals in the input files, but the encoding needs to specified at parsing time and none of the beancount command line tools allow to do that. In practice using anything that is not UTF8 is not supported.